src/app/parcels/ActionBar/ActionModals/DayOverviewModal.tsx

```diff
@@ -13,6 +13,8 @@ const getPdfErrorMessage = (error: DayOverviewPdfError): string => {
         case "parcelFetchFailed":
             errorMessage = "Failed to fetch parcel data.";
             break;
+        case "failedToRetrieveCongestionChargeDetails":
+            errorMessage = "Failed to retrieve congestion charge data";
```

Comment:
should there be a break statement after this? (i think it's not strictly needed for the last case, but we should defo be consistent and help out any future devs who might add more cases below this one!)

Reply:
Done